eng/Versions.props

```diff
@@ -49,8 +49,8 @@
   </PropertyGroup>
   <PropertyGroup>
     <!-- Dependencies from https://github.com/dotnet/sdk -->
-    <MicrosoftNETSdkPackageVersion>5.0.100-preview.5.20225.5</MicrosoftNETSdkPackageVersion>
-    <MicrosoftDotNetMSBuildSdkResolverPackageVersion>5.0.100-preview.5.20225.5</MicrosoftDotNetMSBuildSdkResolverPackageVersion>
+    <MicrosoftNETSdkPackageVersion>5.0.100-preview.5.20230.8</MicrosoftNETSdkPackageVersion>
+    <MicrosoftDotNetMSBuildSdkResolverPackageVersion>5.0.100-preview.5.20230.8</MicrosoftDotNetMSBuildSdkResolverPackageVersion>
```

Comment:
Which version goes into our global.json file -- `5.0.100-preview.5.20230.8` or the package version in the dotnet/installer official build after this gets merged?